tests/object__nongi.js

```diff
@@ -0,0 +1,15 @@
+const gi = require('../lib')
+const Gst = gi.require('Gst', '1.0')
+const { expect } = require('./__common__.js')
+
+gi.startLoop()
+Gst.init()
+
+const bin = new Gst.Bin('name')
+expect(bin.constructor.name, 'GstBin')
+expect(typeof bin.add, 'function')
+console.log(bin.numchildren)
+
+const src = Gst.ElementFactory.make('videotestsrc')
+expect(src.constructor.name, 'GstVideoTestSrc')
+expect(typeof src.addPad, 'function')
```

Comment:
A few adjustements:
 - filename could be `object__non-introspected.js`, no need to abbreviate here
 - wrap the test in a `describe(testDescription, () => ...)`
 - I don't insist on it but testing with `instanceof` could be more interesting than with string testing
 - it could be nice to test that `src` is also an `instanceof` all the parent classes of `GstVideoTestSrc`

Reply:
I added some test with `instanceof`. The problem for non-introspected objects is that we have no access to the corresponding base classes/constructors, do we?